src/components/NavigationSide/navigation-side.css

```diff
@@ -0,0 +1,51 @@
+.sidebar {
+  top: 0;
+  width: 15rem;
+  position: fixed;
+  height: 100%;
+}
+
+.sidebar .sidebar-links {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  margin-top: 50%;
+}
+
+.App.dark .sidebar {
+    background-color: var(--vid-dark-grey);
+}
+
+.App.light .sidebar {
+    background-color: var(--vid-secondary-light);
+}
+
+.sidebar a {
+  display: flex;
+  align-items: center;
+  color: var(--theme-light-font);
+  padding: 1rem 2rem;
```

Comment:
use color name , its easy to interpret color used by dev.
For Eg: theme-light-blue-font